packages/examples/src/remoteCallExample/remoteCall.js

```diff
@@ -50,6 +59,8 @@ window.addEventListener('DOMContentLoaded', (event) => {
             reference: remoteService,
           },
         ],
+        seedAddress: generateAddress(8000),
+        address: generateAddress(1234),
```

Comment:
in createDiscovery add validation on unequality of seedAddress and address (seedAddres !== address)